### ISSUE TITLE
Chemical Imbalance Causes Permanent Very Cold Status

### ISSUE THREAD

**Issue by Grubleafeater** [CleverRaven/Cataclysm-DDA#37439]
# Describe the bug
If you take the Chemical Imbalance trait, some of the 'you feel very cold' triggers cause you to be Very Cold. While the opposite effect fades, leaving you at the temperature you were at, the display for Very Cold continues to be...very cold, with a permanent (Rising!) next to it.

# Steps To Reproduce

Select the Chemical Imbalance trait and wait for Very Cold to trigger

# Expected behavior

Temperature should stabilize after a while.

**Comment by scorpion451** [comment]
Probably related: I've yet to catch it happening in an easy-to-reproduce way, but I've twice had characters experience runaway (as in, +270F) subjective hot temperature when warming from a biotic generator intersects with suddenly feeling hot.

**Comment by jfrikker** [comment]
This happened in my game. It looks like the problem was that the chemical imbalance trait was reducing the temperature of my eyes, but they weren't being warmed back up? I'm not sure if temperature affects the eyes normally (I guess not). I used the debug menu to remove the trait, which didn't help. I also edited the save file by hand to change the eye temperature from where it was (2000) to "normal" (5000). That resolved the Very Cold status issue.
